### DISCUSSION TITLE
exact-time-telegram

### DISCUSSION THREAD

**Discussion by giscus** [AMD-NICK/blog.amd-nick.me#35] [Blog comments]
# exact-time-telegram

Без какой либо на то причины захотелось узнать время отправки сообщения в телеграме с точностью до секунд. Я даже не представляю кому это может быть нужно. Если такие есть - напишите в комментариях - зачем?

https://blog.amd-nick.me/exact-time-telegram

**Comment by dan1pro** [comment]
О, спасибо! Мы с другом спорили кто быстрее отправил сообщение.. и он делает скриншот, у него я первее. А у меня на скриншоте он первее.. ну мы решили как-то проверить ахах)) без тебя бы так и не разрешился бы спор)
